src/lib.rs

```diff
@@ -331,6 +331,9 @@ impl OptLevel {
 
 #[derive(Clone, Debug)]
 pub struct CompileOpts {
+  /// The Bend command
+  pub command: String,
```

Comment:
Instead of storing the command, which is a CLI thing, we should store the target architecture.
For now we can use a structured format, like an enum
```
enum CompileTarget {
  C,
  Cuda,
  Unknown,
}
```